evm/src/main/java/org/hyperledger/besu/evm/precompile/P256VerifyPrecompiledContract.java

```diff
@@ -41,6 +43,7 @@ public class P256VerifyPrecompiledContract extends AbstractPrecompiledContract {
   private static final Bytes32 VALID = Bytes32.leftPad(Bytes.of(1), (byte) 0);
   private static final Bytes INVALID = Bytes.EMPTY;
   private static final int SECP256R1_INPUT_LENGTH = 160;
+  private static final SignatureAlgorithm DEFAULT_SIGNATURE_ALGORITHM_INSTANCE;
```

Comment:
nit: even though this is locally scoped, since the overall besu default is K1, think it's less confusing to call it SECP256R1_SIGNATURE_ALGORITHM_INSTANCE

```suggestion
  private static final SignatureAlgorithm SECP256R1_SIGNATURE_ALGORITHM_INSTANCE;
```

This precompile is also R1-specific so doesn't make sense for anything to override this default.